lib/cli/kit/ini.rb

```diff
@@ -15,8 +15,12 @@ module Kit
     class Ini
       attr_accessor :ini
 
-      def initialize(path = nil, default_section: nil, convert_types: true)
-        @config = File.readlines(path) if path && File.exist?(path)
+      def initialize(path, config: nil, default_section: nil, convert_types: true)
```

Comment:
```suggestion
      def initialize(path = nil, config: nil, default_section: nil, convert_types: true)
```
This should remain nullable, no?

Reply:
I did not realize ruby was smart enough not to make the keywords the first arg if it is nullable.